TopSQL is unavailable caused by panic

```
[2022/10/18 08:02:47.357 +00:00] [ERROR] [misc.go:21] ["panic in the recoverable goroutine"] [r="\"invalid memory address or nil pointer dereference\""] ["stack trace"="github.com/pingcap/ng-monitoring/utils.GoWithRecovery.func1\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/utils/misc.go:23\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:884\nruntime.panicmem\n\t/usr/local/go/src/runtime/panic.go:260\nruntime.sigpanic\n\t/usr/local/go/src/runtime/signal_unix.go:835\ngithub.com/pingcap/ng-monitoring/config/pdvariable.(*variableLoader).loadGlobalConfigLoop\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/config/pdvariable/pdvariable.go:108\ngithub.com/pingcap/ng-monitoring/config/pdvariable.(*variableLoader).start\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/config/pdvariable/pdvariable.go:65\ngithub.com/pingcap/ng-monitoring/utils.GoWithRecovery\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/utils/misc.go:26"] [stack="github.com/pingcap/ng-monitoring/utils.GoWithRecovery.func1\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/utils/misc.go:21\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:884\nruntime.panicmem\n\t/usr/local/go/src/runtime/panic.go:260\nruntime.sigpanic\n\t/usr/local/go/src/runtime/signal_unix.go:835\ngithub.com/pingcap/ng-monitoring/config/pdvariable.(*variableLoader).loadGlobalConfigLoop\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/config/pdvariable/pdvariable.go:108\ngithub.com/pingcap/ng-monitoring/config/pdvariable.(*variableLoader).start\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/config/pdvariable/pdvariable.go:65\ngithub.com/pingcap/ng-monitoring/utils.GoWithRecovery\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/ng-monitoring/utils/misc.go:26"]
```

--- a/config/pdvariable/pdvariable.go
+++ b/config/pdvariable/pdvariable.go
@@ -101,7 +101,7 @@ func (l *variableLoader) loadGlobalConfigLoop(ctx context.Context) {
 			return
 		case <-ticker.C:
 			newCfg, err := l.loadAllGlobalConfig(ctx)
-			if err != nil {
+			if err != nil || newCfg == nil {
 				log.Error("load global config failed", zap.Error(err))
 			} else if cfg != nil {
 				if newCfg != cfg {
@@ -152,7 +152,8 @@ func (l *variableLoader) loadAllGlobalConfig(ctx context.Context) (*PDVariable, 
 			return nil, ctx.Err()
 		default:
 		}
-		etcdCli, err := l.do.GetEtcdClient()
+		var etcdCli *clientv3.Client
+		etcdCli, err = l.do.GetEtcdClient()
 		if err != nil {
 			return nil, err
 		}

